101-rds-deployment-full-ha/azuredeploy.json

```diff
@@ -0,0 +1,1180 @@
+            "location": "[parameters('location')]",
+            "name": "[concat(parameters('projectName'),'intlb')]",
+            "sku": {
+                "name": "Standard"
+            },
+            "dependsOn": [
+                "[concat(parameters('projectName'),'vnet')]"
+            ],
+            "properties": {
+                "frontendIPConfigurations": [
+                    {
+                        "name": "rds-brokers-frontend",
+                        "properties": {
+                            "privateIPAllocationMethod": "Static",
+                            "privateIPAddress": "[concat(parameters('vNetPrefix'),'.0.4')]",
+                            "privateIPAddressVersion": "IPv4",
+                            "subnet": {
+                                "id": "[variables('subNetRef')]"
+                            }
+                        }
+                    },
+                    {
+                        "name": "rds-webgateways-frontend",
+                        "properties": {
+                            "privateIPAllocationMethod": "Dynamic",
+                            "privateIPAddressVersion": "IPv4",
+                            "subnet": {
+                                "id": "[variables('subNetRef')]"
+                            }
+                        }
+                    }
+                ],
+                "backendAddressPools": [
+                    {
+                        "name": "rds-brokers-int-pool",
+                        "properties": {
+                        }
+                    },
+                    {
+                        "name": "rds-webgateways-int-pool",
+                        "properties": {
+                        }                
+                    }
+                ],
+                "probes": [
+                    {
+                        "name": "rds-broker-probe",
+                        "properties": {
+                            "intervalInSeconds": 5,
+                            "numberOfProbes": 2,
+                            "protocol": "Tcp",
+                            "port": 3389
+                        }
+                    },
+                    {
+                        "name": "rds-webgateway-probe",
+                        "properties": {
+                            "intervalInSeconds": 5,
+                            "numberOfProbes": 2,
+                            "protocol": "Tcp",
+                            "port": 443
+                        }
+                    }
+                ],
+                "loadBalancingRules": [
+                    {
+                        "name": "rds-brokers-tcp-rule",
+                        "properties": {
+                            "frontendIPConfiguration": {
+                                "id": "[concat(resourceId('Microsoft.Network/loadBalancers',concat(parameters('projectName'),'intlb')),'/frontendIpConfigurations/rds-brokers-frontend')]"

```

Comment:
remove concat from all the resourceId properties (see the CI results for the full list if needed), e.g.

resourceId('Microsoft.Network/loadBalancers/frontendIpConfigurations', concat(parameters('projectName'),'intlb')),'rds-brokers-frontend')